django_fsm_log/models.py

```diff
@@ -17,7 +17,8 @@ class StateLog(models.Model):
     timestamp = models.DateTimeField(default=now)
     by = models.ForeignKey(getattr(settings, 'AUTH_USER_MODEL', 'auth.User'), blank=True,
                            null=True, on_delete=models.SET_NULL)
-    state = models.CharField(max_length=255, db_index=True)
+    source_state = models.CharField(max_length=255, db_index=True)
```

Comment:
Shouldn't this be `null=True, default=None`?

Reply:
Ouch - I managed to not push the commit. Sorry about that!